src/test/java/io/strimzi/kafka/bridge/http/SeekTest.java

```diff
@@ -0,0 +1,353 @@
+        context.completeNow();
+        assertTrue(context.awaitCompletion(TEST_TIMEOUT, TimeUnit.SECONDS));
+    }
+
+    @Test
+    void seekToEndAndReceive(VertxTestContext context) throws InterruptedException, ExecutionException, TimeoutException {
+        String topic = "seekToEndAndReceive";
+        kafkaCluster.createTopic(topic, 1, 1);
+
+        JsonObject topics = new JsonObject();
+        topics.put("topics", new JsonArray().add(topic));
+
+        JsonObject jsonConsumer = new JsonObject();
+        jsonConsumer.put("name", name);
+        // create consumer
+        // subscribe to a topic
+        consumerService()
+            .createConsumer(context, groupId, jsonConsumer)
+            .subscribeConsumer(context, groupId, name, topic);
+
+        CompletableFuture<Boolean> dummy = new CompletableFuture<>();
+        // dummy poll for having re-balancing starting
+        consumerService()
+            .consumeRecordsRequest(groupId, name, BridgeContentType.KAFKA_JSON_BINARY)
+                .as(BodyCodec.jsonArray())
+                .send(ar -> {
+                    context.verify(() -> assertTrue(ar.succeeded()));
+                    dummy.complete(true);
+                });
+
+        dummy.get(TEST_TIMEOUT, TimeUnit.SECONDS);
+
+        kafkaCluster.produceStrings(topic, 10, 0);
+
+        // seek
+        JsonArray partitions = new JsonArray();
+        partitions.add(new JsonObject().put("topic", topic).put("partition", 0));
+
+        JsonObject root = new JsonObject();
+        root.put("partitions", partitions);
+
+        CompletableFuture<Boolean> seek = new CompletableFuture<>();
+        seekService()
+            .positionsBeginningEnd(groupId, name, root)
+                .sendJsonObject(root, ar -> {
+                    context.verify(() -> {
+                        assertTrue(ar.succeeded());
+                        assertEquals(HttpResponseStatus.NO_CONTENT.code(), ar.result().statusCode());
+                    });
+                    seek.complete(true);
+                });
+
+        seek.get(TEST_TIMEOUT, TimeUnit.SECONDS);
+
+        CompletableFuture<Boolean> consumeSeek = new CompletableFuture<>();
+        // consume records
+        baseService()
+            .getRequest(Urls.consumerInstanceRecords(groupId, name))
+                .putHeader(ACCEPT, BridgeContentType.KAFKA_JSON_BINARY)
+                .as(BodyCodec.jsonArray())
+                .send(ar -> {
+                    context.verify(() -> assertTrue(ar.succeeded()));
+
+                    JsonArray body = ar.result().body();
+                    assertEquals(0, body.size());
+                    consumeSeek.complete(true);
+                });
+
+        consumeSeek.get(TEST_TIMEOUT, TimeUnit.SECONDS);
+
+        CompletableFuture<Boolean> delete = new CompletableFuture<>();
+        // consumer deletion
+        consumerService()
+            .deleteRequest(Urls.consumerInstance(groupId, name))
```

Comment:
if we have to delete the consumer, why not using the `consumerService().deleteConsumer(context, groupId, name);` as for all the other consumer tests?